distribution/changelog.txt

```diff
@@ -17,6 +17,7 @@
 - Fix: [#13074] Entrance and exit ghosts for mazes not being removed.
 - Fix: [#13083] Dialog for renaming conflicting track design crops text out.
 - Improved: [#13023] Made add_news_item console command last argument, assoc, optional.
+- Fix: [#13125] Support localisation for RollerCoaster Tycoon 2 data picker dialogs
```

Comment:
We group by category, and then keep the list ordered by issue ID. Please move this line so the changelog retains it numerical order.